deepchecks/tabular/checks/model_evaluation/train_test_prediction_drift.py

```diff
@@ -46,6 +47,11 @@ class TrainTestPredictionDrift(TrainTestCheck, ReduceMixin):
 
     Parameters
     ----------
+    probability_drift: str, default: 'auto'
+        Controls whether to compute drift on the predicted probabilities or the predicted classes. If 'auto', compute
+        drift on the predicted probabilities if the task is multiclass, and on the predicted classes otherwise.
+        Set to 'true' to force drift on the predicted probabilities, and 'false' to force drift on the predicted
+        classes.
```

Comment:
I think that the name "probability drift" is problematic. Should be something more neutral in my opinion... Also, passing "true" as string is very confusing.
Maybe "drift_mode"? And then the option will be "Auto", "Proba", "Prediction" or something like that?

Reply:
+1

Reply:
👍